crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -661,14 +703,15 @@ async fn bulk_operations(
     reqs: Json<Vec<ContextAction>>,
     db_conn: DbConnection,
     user: User,
+    #[cfg(feature = "high-performance-mode")] tenant: Tenant,
     tenant_config: TenantConfig,
 ) -> superposition::Result<HttpResponse> {
     use contexts::dsl::contexts;
     let DbConnection(mut conn) = db_conn;
     let tags = parse_config_tags(custom_headers.config_tags)?;
-
+    let mut version_id = 0;
```

Comment:
Can we make it optional instead of initiating it with 0?

Reply:
Its not optional though